v2.6/getting-started/docker/installation/requirements.md

```diff
@@ -10,6 +10,10 @@ in order for Calico to function properly with Docker.
 
 As with all Calico clusters, all hosts should have IP connectivity between them.
 
+### iptables
+
+Note that on some systems, iptables runs in nftables compatibility mode by default. Calico v2.6 is not compatible with this and it will likely result in network connectivity issues. To fix this, run the following command to configure iptables to run in legacy mode: `sudo update-alternatives --set iptables /usr/sbin/iptables-legacy`. You will need to reboot for the change to take effect.
```

Comment:
I think that command only works on Debian-based distributions, probably worth saying `On Debian-based systems, you can switch to iptables-legacy mode by running this command...`

Reply:
Added a note to say that the command is debian-based system specific. Sorry about that!